### PR TITLE
Add cluster versions panel to dashboard.

### PR DESCRIPTION
**Pull request by Gerrit91**

### CODE REVIEW

File: cmd/dashboard.go [hunk 1]
```diff
@@ -586,6 +582,324 @@ func (d *dashboardClusterPane) Render() error {
...
+
+	sort.Slice(versions, func(i, j int) bool {
+		if versions[i].parsedVersion == nil && versions[j].parsedVersion == nil {
+			return versions[i].rawVersion < versions[j].rawVersion
+		}
+
+		if versions[i].parsedVersion == nil {
+			return false
+		}
+		if versions[i].parsedVersion == nil {
```
Comment (majst01) [inline cmd/dashboard.go:689]:
```suggestion
		if versions[j].parsedVersion == nil {
```

Comment (majst01) [reply cmd/dashboard.go:689]: Why not:

```
sort.Sort(semver.Collection(slice-of-semvers))
```
?

Comment (Gerrit91) [reply cmd/dashboard.go:689]: It should also sort non-parsable versions by string.

Comment (Gerrit91) [reply cmd/dashboard.go]: Don't know if there is a chance to have something like SHAs in there as we have it for the firewall-controller for instance. Just wanted to make sure we always have a defined sort order. 😅

**Review by majst01 — APPROVED** [review]

**Pull request merged**